spikesorters/sorterlist.py

```diff
@@ -29,7 +29,7 @@
 
 # generic laucnher via function approach
 def run_sorter(sorter_name_or_class, recording, output_folder=None, delete_output_folder=False,
-               grouping_property=None, parallel=False, verbose=False, **params):
+               grouping_property=None, parallel=False, verbose=False, raise_error=True, **params):
```

Comment:
I don't see where you pass this raise_error to the BaseSorter.run(...)

Reply:
I forgot it! added now